.github/dependabot.yml

```diff
@@ -0,0 +1,94 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: weekly
+    day: sunday
+    time: "02:00"
+  open-pull-requests-limit: 30
+  labels:
+  - dependencies
+  ignore:
+  - dependency-name: "@babel/*"
+    versions:
+    - ">= 0"
+  - dependency-name: core-js
+    versions:
+    - ">= 0"
+  - dependency-name: "@date-io/*"
+    versions:
+    - ">= 0"
+  - dependency-name: jss-*
+    versions:
+    - ">= 0"
+  - dependency-name: raw-loader
+    versions:
+    - ">= 0"
+  - dependency-name: react-dom
+    versions:
+    - ">= 0"
+  - dependency-name: react-is
+    versions:
+    - ">= 0"
+  - dependency-name: react-test-renderer
+    versions:
+    - ">= 0"
+  - dependency-name: tslint
+    versions:
+    - 5.x
+  - dependency-name: "@typescript-eslint/parser"
+    versions:
+    - ">= 0"
+  - dependency-name: karma-webpack
+    versions:
+    - 5.0.0
+    - 5.0.0
+    - 5.0.0
+  - dependency-name: webpack
+    versions:
+    - 5.19.0
+    - 5.21.1
+    - 5.21.2
+    - 5.23.0
+    - 5.24.2
+    - 5.24.3
+    - 5.25.0
+    - 5.27.1
+    - 5.31.2
+    - 5.35.1
```

Comment:
I hope this doesn't mean that everytime we close a PR the file gets updated.

I'll remove these for now to get parity with the old config and then we'll see if we need them.